src/helpers.js

```diff
@@ -469,7 +467,8 @@ export function createStreamInfo(item, mediaSource, startPosition) {
         console.log('Subtitle url: ' + info.subtitleStreamUrl);
     });
 
-    if (subtitleTracks) {
+    if (subtitleTracks && subtitleTracks.length) {
```

Comment:
I made this change trying to get the tracks undefined if there were none, but I think I didn't understand how things worked at the time, so this might nt actually do anything useful.
(for reference: `if (subtitleTracks && subtitleTracks.length)`)